tool/generator/lib/src/inputs/interfaces.dart

```diff
@@ -107,6 +107,7 @@ final windowsRuntimeTypesToGenerate = <String>{
   'Windows.Data.Xml.Dom.IXmlNodeList',
   'Windows.Foundation.IAsyncInfo',
   'Windows.Foundation.IClosable',
+  'Windows.Foundation.IPropertyValue',
```

Comment:
IPropertyValue is an outlier as PropertyValue doesn't depend on it (it is not generated automatically), so we need to add this manually.

Reply:
That's most odd, isn't it. I see that now. `PropertyValue` is a static class. Thanks.